chargeio/infrastructure/ChargeIOError.cs

```diff
@@ -1,23 +1,24 @@
-﻿using System.Collections.Generic;
-using Newtonsoft.Json;
-
-namespace ChargeIO
-{
-	public class ChargeIOError
-	{
-		[JsonProperty("message")]
-		public string Message { get; set; }
-
-		[JsonProperty("code")]
-		public string Code { get; set; }
-
-		[JsonProperty("level")]
-		public string Level { get; set; }
-
-		[JsonProperty("context")]
-		public string Context { get; set; }
-
-    	[JsonProperty("sub_code")]
-		public string SubCode { get; set; }
-	}
+﻿using System;
+using Newtonsoft.Json;
+
+namespace chargeio

```

Comment:
What is the reason for the namespace change from ChargeIO to chargeio throughout? If necessary, fine, but if not this only seems to make upgrading more difficult for consumers.